docs/get-started.md

```diff
@@ -4,14 +4,43 @@ title: Temporal documentation
 sidebar_label: Get started
 ---
 
+Welcome to Temporal's user documentation!
+
 Temporal reimagines state-dependent service-orchestrated application development, and so much more.
 
 ## Get started
 
-This is page will be dedicated to directing users on how to get started with Temporal.
+Ready to develop? [Install the Temporal server](install-temporal-server) and choose your SDK.
+
+→ [Go SDK](go-quick-start)
```

Comment:
So visually speaking the arrows are much more aesthetic. That being said, we also use bullets on the page and I would like to establish when we will use bullets vs arrows.

Reply:
I too prefer the arrows - but wanted to see what you thought first.
As of right now, arrows need to be manually added via a copy and paste into the doc. They do not automatically format into lists - 
So, as a long term solution what we will want to do is update the custom css to make markdown `<li>` tags render as those arrows vs the bullets. I will introduce that change in another pr and convert all of these to regular list items for now.